yarn.lock

```diff
@@ -1922,9 +1728,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@esbuild/aix-ppc64@npm:0.24.0":
-  version: 0.24.0
-  resolution: "@esbuild/aix-ppc64@npm:0.24.0"
+"@esbuild/aix-ppc64@npm:0.25.2":
```

Comment:
All esbuild deps have duplicates

Reply:
That's probably because we still use vite 5. Upgrading `vite` and `tsx` should target the same `esbuild` as storybook:
- https://github.com/vitejs/vite/blob/v6.2.4/packages/vite/package.json
- https://github.com/privatenumber/tsx/releases/tag/v4.19.3

![image](https://github.com/user-attachments/assets/2b558e2c-47b6-478a-bb56-db6f55e443d7)

Reply:
We decided to tackle the other dependencies in another PR